conferencias4/src/screens/InvitePeople.js

```diff
@@ -3,7 +3,7 @@ import { StyleSheet, View,Text,TextInput,FlatList} from 'react-native';
 import { SafeAreaProvider,useSafeArea } from 'react-native-safe-area-context';
 import { List } from 'react-native-paper';
 import FabButton from "../components/FabButton" 
-import SelectDate from "../screens/SelectDate"
+
 
 
 
```

Comment:
When you use FlatList, in your array 'people' you need to put a key (is the same as an id) or you'll have a warning